data/json/items/armor.json

```
@@ -6465,7 +6465,7 @@
     "name": "dust mask",
     "description": "A simple piece of cotton that straps over the mouth.  Provides a small amount of protection from airborne illness and dust.",
     "weight": 168,
-    "volume": 0,
+    "volume": "0 L",
```

```suggestion
    "volume": "100 ml",
```
Going off of [a mask of these dimensions](https://www.amazon.com/dp/B07MTV8XCQ/ref=psdc_2257619011_t1_B07MNW7ZTB), 3.9 x 2.8 x 2.8 inches, then multiplying by 0.2, because these are far from solid rectangles.